vmdb/lib/miq_automation_engine/service_models/miq_ae_service_orchestration_template_cfn.rb

```diff
@@ -0,0 +1,3 @@
+module MiqAeMethodService
+  class MiqAeServiceOrchestrationTemplateCfn < MiqAeServiceOrchestrationTemplate; end
```

Comment:
In the name `MiqAeServiceOrchestrationTemplateCfn`, what does `Cfn` suffix mean?